wiki/content/slash-graphql/admin/one-click-deploy.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "One-click Deploy"
+[menu.main]
+    parent = "slash-graphql-admin"
+    weight = 5
++++
+
+It is possible to deploy a new instance of the sample apps built in Slash GraphQL. This will spin a fresh backend with a new instance of the sample app hosted and schema generated for you.
```

Comment:
We love building GraphQL apps at Dgraph! In order to help you get started quickly, we are making it easy to launch a few apps with a single click. We'll create a fresh backend for you, load it with the right schema, and even launch a front end that's ready for you to use.